### PR TITLE
Bump ImageSharp.Drawing to 1.0.0

### PR DESCRIPTION
**Pull request by VisualMelon**
I was going to refit ImageSharp completely, but then I realised that ImageSharp.Drawing is fine with version 1.0.0 still, so this just bumps it so that it's not a beta version.

### Checklist

- [ ] I have included examples or tests
- [x] I have updated the change log
- [x] I am listed in the CONTRIBUTORS file
- [ ] I have cleaned up the commit history (use rebase and squash)

### Changes

 - Bump SixLabors.ImageSharp.Drawing dependencies to version 1.0.0
 - Add support for LineJoin in OxyPlot.ImageSharp/ImageRenderContext
 - Enable nullable in OxyPlot.ImageSharp

@oxyplot/admins

### CODE REVIEW

**Comment by VisualMelon** [comment]
TODO:
 - import fixes from #2098 where applicable (zero thickness crash)
 - see if we can get line-join to work

**Comment by VisualMelon** [comment]
TODO bits are done and tests all run; some examples are erroring but not sure if that's new.

**Comment by VisualMelon** [comment]
Need to think about why `fontFamily` is nullable in `IRenderContext`.

**Comment by VisualMelon** [comment]
Only errors that seem to be generated now are related to missing fonts

File: Source/OxyPlot.ImageSharp/ImageRenderContext.cs [hunk 1]
```diff
@@ -254,9 +258,10 @@ public override void DrawText(ScreenPoint p, string text, OxyColor fill, string
         }
 
         /// <inheritdoc/>
-        public override OxySize MeasureText(string text, string fontFamily = null, double fontSize = 10, double fontWeight = 500)
+        public override OxySize MeasureText(string text, string? fontFamily = null, double fontSize = 10, double fontWeight = 500)
         {
-            return this.MeasureTextLoose(text, fontFamily, fontSize, fontWeight);
+            // TODO: should fontFamily be nullable?
```
Comment (Jonarw) [inline Source/OxyPlot.ImageSharp/ImageRenderContext.cs]: Were you able to come to a conclusion on this?

**Review by Jonarw — COMMENTED** [review]
Other than the one TODO, looks good to me!

File: Source/OxyPlot.ImageSharp/ImageRenderContext.cs (hunk 1 above)
Comment (VisualMelon) [reply Source/OxyPlot.ImageSharp/ImageRenderContext.cs]: The `RenderErrorMessage` logic leaves it `null`, which I presume is intentional to allow the render context to pick a simple font if there are rendering errors, or something like that. The example library is also full of `null` font families, so I don't think we can change it in a hurry. I did spot a bug, though, which I've lazily pushed as a small addition to this PR.

**Review by Jonarw — APPROVED** [review]

**Pull request merged**